src/main/java/com/bina/varsim/types/variant/Variant.java

```diff
@@ -550,6 +557,15 @@ public boolean hasCN() {
     public StringBuilder alt_string() {
         StringBuilder sbStr = new StringBuilder();
         for (int i = 0; i < _alts.length; i++) {
+            if (i > 0 && _alts[i].toString().equals(_alts[i - 1].toString())) {
+//               if two alternative alleles are the same (e.g. two symbolic alleles)
```

Comment:
Right now the VCF parser does not correctly parse two alt symbolic alleles, shall I change the parser so that it accepts two alt symbolic alleles? It does honor two values in SVLEN field, though.